fennel/client/client.py

```diff
@@ -813,6 +844,21 @@ def _post(
         check_response(response)
         return response
 
+    @staticmethod
+    def _parse_dataframe(
```

Comment:
Can you add a description to this function.
Also where is it supposed to be used ?

Reply:
added